### PR TITLE
comp: badges and buttons added with new designs

### PR DESCRIPTION
**Pull request by manojsarna**

### CODE REVIEW

**Review by PuneetP16 — APPROVED** [review]
Nice clean and consistent code.

**Comment by rahulyadav139** [comment]
Code is readable and clean. Animations are subtle. Great work.

File: lib/Components/Badges/badges.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
+@import "/lib/Components/Avatar/avatar.css";
+
+.avatar-badge{
+    overflow: initial;
+}
+
+.av-badge{
```
Comment (mehulsatardekar) [inline lib/Components/Badges/badges.css:7]: The naming convention can be improved

**Review by mehulsatardekar — CHANGES_REQUESTED** [review]

**Comment by mehulsatardekar** [comment]
Other than that code is readable and UI is elegant

File: lib/Components/Alerts/alerts.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+.alert-info{
+    
+    background-color: var( --COLOR-INFO-LIGHT);
+    color: var(--COLOR-INFO);
+    border: 1px solid var(--COLOR-INFO);
+    border-top-width: 6px;
+}
+
+
+
```
Comment (iRohitGaur) [inline lib/Components/Alerts/alerts.css:21]: suggestion: keep consistency in spacing

File: lib/Components/Alerts/alerts.css [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    border: 1px solid var(--COLOR-ERROR);
+    border-top-width: 6px;
+}
+
+
+
+.alert-icon{
+    margin: 1rem;
+    /* width:2rem;
+    height: 2rem; */
```
Comment (iRohitGaur) [inline lib/Components/Alerts/alerts.css:48]: remove commented code

File: lib/Components/Badges/badges.css [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    font-weight: 500;
+    top:-10px;
+    right:-12px;
+}
+
+.bd-btn{
+    outline: none;
+    cursor: pointer;
+    font-weight: 600;
+    padding: 0.5rem;
```
Comment (iRohitGaur) [inline lib/Components/Badges/badges.css]: suggestion: you can use px value for less than 10px

**Review by iRohitGaur — APPROVED** [review]
Great work! Good to merge 👍🏻

**Comment by m26unkwn** [comment]
great work Manoj buttons are so nice.

**Review by m26unkwn — APPROVED** [review]

**Comment by Ramandeep9898** [comment]
loved that green color... and images :0

File: lib/Components/Alerts/alerts.js [hunk 5]
```diff
@@ -7,3 +7,8 @@ alertCloseBtn.addEventListener("click",()=>{
     alertToRemove.style.display = "none";
 })
 
+// const btnID = document.querySelector("#btnID");
+
+// console.log(btnID);
+
+// btnID.addEventListener("click",()=>console.log("clicked"));
```
Comment (saurabhchirde) [inline lib/Components/Alerts/alerts.js:14]: Remove commented code.

**Review by saurabhchirde — APPROVED** [review]
Everything looks good to merge.

**Pull request merged**